src/Autofac.Configuration/Autofac.Configuration.csproj

```diff
@@ -18,10 +19,8 @@
     <PackageLicenseExpression>MIT</PackageLicenseExpression>
     <RepositoryType>git</RepositoryType>
     <RepositoryUrl>https://github.com/autofac/Autofac.Configuration</RepositoryUrl>
-    <GenerateAssemblyTitleAttribute>false</GenerateAssemblyTitleAttribute>
-    <GenerateNeutralResourcesLanguageAttribute>false</GenerateNeutralResourcesLanguageAttribute>
-    <GenerateAssemblyCopyrightAttribute>false</GenerateAssemblyCopyrightAttribute>
-    <GenerateAssemblyDescriptionAttribute>false</GenerateAssemblyDescriptionAttribute>
+    <AssemblyTitle>Autofac.Configuration</AssemblyTitle>
+    <NeutralLanguage>en-GB</NeutralLanguage>
```

Comment:
All the other assemblies default to `en-US`.

Reply:
(Nice try, buddy!) 😆

Reply:
🙄